Refactor processing of named graphs provided by the TCK

boggle opened the pull request.

boggle committed:
Refactor processing of named graphs provided by the TCK

boggle commented:
Pending https://github.com/opencypher/openCypher/pull/78

Mats-SX commented on community/cypher/compatibility-suite/src/test/java/cypher/FeatureSuiteTest.java:
```diff
@@ -23,15 +23,15 @@
     // If you want to run only a single scenario, put (part of) its name in the `SCENARIO_NAME_REQUIRED` constant
     // Do not forget to clear these strings to empty strings before you commit!!
-    public static final String FEATURE_TO_RUN = "";
+    public static final String FEATURE_TO_RUN = "CineastsDependent.feature";
```
Nope!

Mats-SX commented:
78 was closed; this is now pending https://github.com/opencypher/openCypher/pull/79

boggle committed:
Update to latest openCypher TCK version

o Also remove use of parameters from named graphs
o Changed to using '.cypher' extension for importing named graphs
o Blacklisted some scenarios broken by recent changes

boggle committed:
Removed old solution for handling named graphs

Mats-SX commented on community/cypher/compatibility-suite/src/test/java/cypher/UnpackedResources.java:
```diff
@@ -85,8 +85,9 @@ private static void findAndUnpackTo( FileSystem sourceFileSystem, String sourceP
-//                Files.copy( next, target.toPath(), StandardCopyOption.REPLACE_EXISTING );
-                Files.copy( next, target.toPath() );
+                Files.copy( next, target.toPath()
+                        , StandardCopyOption.REPLACE_EXISTING
```
I don't think this should be the default.
